client/blocks/jitm/style.scss

```diff
@@ -5,7 +5,22 @@
 	max-width: 1040px;
 }
 
+// Customization for the /themes page.
 .is-section-themes-i4 .layout__content > .upsell-nudge {
 	max-width: 100%;
 	margin: 40px 0 0 0;
 }
+
+// Customization for the /stats page.
+.is-section-stats .layout__content > .upsell-nudge {
+	max-width: 1224px;
```

Comment:
Does it make sense to use variables instead, like `$stats-sections-max-width` for 1224 and `$custom-mobile-breakpoint` for 660?

Reply:
Probably so. 😅 I'm going to go ahead and merge this as is to get this in before I go AFK. Good idea for the next update!